src/engine/matrix.hpp

```diff
@@ -1,13 +1,13 @@
 #pragma once
 #include "vec.hpp"
 
-/** Template 2x2 matrix class */
+/** Template 2row x 2col matrix class */
```

Comment:
I meant is this column major or row major, i.e. when you're storing a list of these numbers: `[1, 2, 3, 4]` is that a matrix which is:
```
|1 2|
|3 4|
or
|1 3|
|2 4|
```